include/arkode/arkode.h

```diff
@@ -214,6 +214,10 @@ typedef enum
  * Shared API routines
  * -------------------------- */
 
+/* Command-line control over ARKODE options */
+SUNDIALS_EXPORT int ARKodeSetFromCommandLine(void* arkode_mem, const char* arkid,
+                                             int argc, char* argv[]);
```

Comment:
```suggestion
SUNDIALS_EXPORT int ARKodeSetFromCommandLine(void* arkode_mem, const char* arkid,
                                             int argc, const char* argv[]);
```
I'm not sure if I suggested this already... Functions with this arguments will also need to be updated.

Reply:
You did suggest this previously, but I could not get it to work.  If I recall the error messages correctly, the problem is that this it is essentially a `char**` so the compiler cannot ensure that functions will not interfere with its internal memory.  I ended up having to propagate the `const` all the way to the `main` routine for it to compile, and I believe that is overly restrictive for user code (who may want to remove command-line options after they are parsed).